src/core/packages/overlays/browser-internal/src/flyout/flyout_service.test.tsx

```diff
@@ -56,7 +56,7 @@ describe('FlyoutService', () => {
     it('renders a flyout to the DOM', () => {
       expect(mockReactDomRender).not.toHaveBeenCalled();
       flyouts.open(mountText('Flyout content'));
-      expect(mockReactDomRender.mock.calls).toMatchSnapshot();
+      expect(JSON.stringify(mockReactDomRender.mock.calls)).toMatchSnapshot();
```

Comment:
Need to serialize this object before checking the snapshot, to avoid the error, `TypeError: Cannot delete property 'css' of #<Object>`